doc/modules/language-guide/pages/language-manual.adoc

```diff
@@ -657,10 +665,10 @@ The type `Bool` of category L (Logical) has values `true` and `false` and is sup
 === Type `Char`
 
 A `Char` of category O (Ordered) represents characters as a code point in the Unicode character
-set. Characters can be converted to `Nat32`, and `Nat32`s in the
-range *0 .. 0x1FFFFF* can be converted to `Char` (the conversion traps
-if outside of this range). With `charToText` a character can be
-converted into a text of length 1.
+set. Characters can be converted to `Nat32`, and ``Nat32``s in the
+range _0 .. 0x1FFFFF_ can be converted to `Char` (the conversion traps
```

Comment:
The upper limit for a unicode scalar value actually is 0x10FFFF. Also, 0xD800 .. 0xDFFFF are not legal. (I just checked, Char.fromNat32 correctly traps for anything outside that.)

Reply:
Fixed.